editor/components/post-publish-panel/toggle.js

```diff
@@ -33,17 +46,31 @@ function PostPublishPanelToggle( { isSaving, isPublishable, isSaveable, isPublis
 			disabled={ ! isButtonEnabled }
 			isBusy={ isSaving && isPublished }
 		>
-			<PostPublishButtonLabel />
-			<Dashicon icon="arrow-down" />
+			<PostPublishButtonLabel />...
```

Comment:
Unfortunately appending the ellipsis like this won't work for RTL languages such as Hebrew.

Perhaps we could pass a prop to `PostPublishButtonLabel` and change the localised string accordingly:

```javascript
// editor/components/post-publish-panel/toggle.js:49
<PostPublishButtonLabel showEllipsis={ true } />

// editor/components/post-publish-button/label.js:50
return showEllipsis ? __( 'Publish…' ) : __( 'Publish' );
```

Reply:
I guess we already checked that we're publishing here so we might just replace the component with `__( 'Publish…' )`